nixos/tests/os-prober.nix

```diff
@@ -8,7 +8,7 @@ let
       ${parted}/bin/parted --script /dev/vda mklabel msdos
       ${parted}/sbin/parted --script /dev/vda -- mkpart primary ext2 1M -1s
       mkdir /mnt
-      ${e2fsprogs}/bin/mkfs.ext4 /dev/vda1
```

Comment:
what does that do? (Out of curiosity)

Reply:
I don't know, except that mkfs.ext4 didn't used to enable it by default (or support it at all), and now it does.